src/app/shared/utils/file-size-pipe.ts

```diff
@@ -12,30 +13,7 @@ import { Pipe, PipeTransform } from '@angular/core';
 
 @Pipe({ name: 'dsFileSize' })
 export class FileSizePipe implements PipeTransform {
-
-  private units: string[] = [
-    'bytes',
-    'KiB',
-    'MiB',
-    'GiB',
-    'TiB',
-    'PiB'
-  ];
-
   transform(bytes: number = 0, precision: number = 2): string {
-    let result: string;
-    if (isNaN(parseFloat(String(bytes))) || !isFinite(bytes)) {
-      result = '?';
-    } else {
-      let unit = 0;
-
-      while (bytes >= 1024) {
-        bytes /= 1024;
-        unit++;
-      }
-
-      result = bytes.toFixed(+ precision) + ' ' + this.units[unit];
-    }
-    return result;
+    return fileSize(bytes, { standard: 'iec', round: precision });
```

Comment:
I think we want to be using the default `standard` here, which is JEDEC.  "IEC" standard says there's 1000 bytes in a kilobyte (and uses the abbreviation KiB), whereas JEDEC says there's 1024 bytes in a KB.  Our prior code obviously used 1024, and the KB abbreviation.

Reply:
That's not entirely accurate @tdonohue. Both say there are 1024 bytes in a kilobyte, as you can see in [this jsfiddle I made to test it with the lib used in this PR](https://jsfiddle.net/1h3oujtL/). The only difference is that IEC uses KiB and JEDEC uses KB.

Currently in master [dspace 7 uses KiB](https://dspace7-demo.atmire.com/items/1974d7d3-9648-4dbe-b083-56374f5314e4/full), and [dspace 6 and below used KB](http://demo.dspace.org/xmlui/handle/10673/5?show=full). I originally chose KiB for dspace 7 because I thought it would convey we were using base 2 for those that knew what the little 'i' meant, and those that didn't most likely wouldn't notice. However I'm not opposed to switching back to JEDEC if you think that's better

Reply:
@artlowel : I'd recommend we use JEDEC as that's what I feel most are more familiar with (and I think that's why it's the default). I'm worried "KiB" honestly will bring up more questions rather than understanding.  So, I'd like this to use KB/MB/GB syntax please.

Reply:
Ok. I'll ask @Atmire-Kristof to change it

Reply:
Done